### PR TITLE
feat: add bulk offline donations postman collection and associated en…

### PR DESCRIPTION
**Pull request by jfostner**
…vironments

### CODE REVIEW

File: bulk-offline-donations/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Using the Classy API
+
+If you are using your Classy Account, ensure you have your Client ID and Client Secret from Classy Manager. If you don't have these or don't know how to get them, you can find that information from this support article: <https://support.classy.org/s/article/intro-to-the-classy-api>
+
+Once you have these, ensure you are using the Classy API environment, and enter the Client ID and Client Secret into the 'current value' field for client_id and client_secret respectively.
+Before making any of the requests to fetch source codes, you'll need to authenticate with the API. To do so, use the App Auth request under the Authentication folder. This will make a call to the API and, assuming you have working credentials, will provide a token that will be automatically saved and used for the other requests. These tokens have a limited timeframe in which they are valid, so if you start seeing errors that you have an invalid token, simply run the App Auth request again to get a new token.
+
+Once you have this token, you should be ready to start sending requests to the Classy API and syncing offline donations into your campaigns.
```
Comment (mmills-gfm) [inline bulk-offline-donations/README.md:23]: Do we want to put a disclaimer here that the users need to be careful when they execute a bulk load against their account, and that if they add bad/meaningless data they would need to clean it up themselves? Just want to make sure we don't get upset people coming to us saying they followed the sample app and now they have a bunch of junk data in their account

File: bulk-offline-donations/Bulk Offline Donations Mock Server.postman_environment.json
Comment (rosborn-gfm) [inline bulk-offline-donations/Bulk Offline Donations Mock Server.postman_environment.json:1]: should we remove the username, password, and refresh_token variables? I'm suspecting most users will be authenticating with an app token.

File: bulk-offline-donations/README.md (hunk 1 above)
Comment (jfostner) [reply bulk-offline-donations/README.md:23]: Is this something that we should have Kate write/go over?

Comment (mmills-gfm) [reply bulk-offline-donations/README.md:23]: I think that's a good point, if we can wordsmith something and run it by her that would be great

**Review by rosborn-gfm — APPROVED** [review]

**Pull request merged**